mesonbuild/scripts/scanbuild.py

```diff
@@ -34,7 +35,31 @@ def run(args):
     privdir = os.path.join(blddir, 'meson-private')
     logdir = os.path.join(blddir, 'meson-logs/scanbuild')
     shutil.rmtree(logdir, ignore_errors=True)
-    exename = os.environ.get('SCANBUILD', 'scan-build')
+    tools = [
+        'scan-build-5.0',  # latest stable release
+        'scan-build-4.0',  # old stable releases
+        'scan-build-3.9',
+        'scan-build-3.8',
+        'scan-build-3.7',
+        'scan-build-3.6',
+        'scan-build-3.5',
+        'scan-build',  # base, points to 4.0 or older on Ubuntu
+        'scan-build-6.0',  # development snapshot
```

Comment:
Should we default to this order or is the default value without any number behind it? The comments here are going to get out of sync fairly soon. Maybe it would be better to add a comment above the array describing why the order is what it is and how the array should be updated in the future.

Reply:
I guess the list is copied from `LLVMDependency`. Why move the base (`scan-build`) to nearly bottom of the list, and why not add names used by FreeBSD (`llvm-config50`, `llvm-config40`, `llvm-config-devel` ...) ?